index.html

```diff
@@ -61,6 +61,22 @@ <h1 class="display-3">Bikes for Refugees</h1>
 							<a class="btn btn-secondary btn-lrg" href="#">Volunteer</a>
 						</div>
 					</div>
+					<div class="container my-container ">
+						<div class="row my-row justify-content-between">
+							<div class="box col-md-3 col-12 my-col "><strong>
```

Comment:
`my-col` and `my-row` are not used here, please remove them. 
As an aside, you should not be using classes which a) overwrite the Bootstrap grid classes (`container`, `row` and `col-...`), as this would interfere with the grid's layout; and b) which are named `my-something`, it's not descriptive and when you work on shared code with other people it loses its meaning.

Reply:
Same thing with `my-container`. Be more descriptive.